src/mono/wasm/Wasm.Build.Tests/Blazor/BuildPublishTests.cs

```diff
@@ -24,11 +24,15 @@ public BuildPublishTests(ITestOutputHelper output, SharedBuildPerTestClassFixtur
     }
 
     [Theory, TestCategory("no-workload")]
-    [InlineData("Debug")]
-    [InlineData("Release")]
-    public void DefaultTemplate_WithoutWorkload(string config)
+    [InlineData("Debug", true)]
```

Comment:
This, and the other test variations are multiplying the number of tests with not a lot of extra value. Instead, I would suggest adding new tests just for this unicode char:

0. Include the unicode char in the project name
1. test with relinking, which can happen in build, or publish
2. Same test with publish+aot